static/templates/template/components/text-edit.html

```diff
@@ -1,21 +1,31 @@
 <ng-include src="'/static/templates/template/components/question-edit.html'"></ng-include>
 
-
-<md-input-container ng-if="item.sub_type=='number'" class="_text-input _tiny-text" md-no-float>
-  <input  type="number" tabindex="0" ng-model="item.answer" ng-required="item.required"
-         aria-label="{{item.name}}" max="{{item.aux_attributes.max}}" min="{{item.aux_attributes.min}}"
-          ng-disabled="true" placeholder="{{item.aux_attributes.placeholder}}">
+<md-input-container ng-if="item.aux_attributes.pattern.type=='number'" class="_text-input _tiny-text" md-no-float>
+  <input type="number" tabindex="0" ng-model="item.answer" ng-required="item.required"
+         aria-label="{{item.name}}" ng-disabled="true" placeholder="{{item.aux_attributes.placeholder}}">
+</md-input-container>
+<md-input-container
+  ng-if="item.sub_type=='text' && item.aux_attributes.pattern.type!='number' && item.aux_attributes.pattern.specification!='email'"
```

Comment:
For edit, we may only need segregation at sub-type level as these are only display fields and remain disabled. We can even remove all attributes as they are not relevant for a disabled field.